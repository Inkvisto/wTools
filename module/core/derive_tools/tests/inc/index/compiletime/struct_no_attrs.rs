use derive_tools::Index;

#[ derive( Index ) ] 
#[ index () ]
struct StructMultipleNamed< T > 
{
  a : Vec< T >,
  b : Vec< T >,
}

fn main()
{  
}

