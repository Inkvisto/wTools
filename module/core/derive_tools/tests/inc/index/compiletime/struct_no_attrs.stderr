error: No attributes specified. You must to specify #[ index ] for fields or name for #[ index ( name = field_name ) ] for item derive macro
 --> tests/inc/index/compiletime/struct_no_attrs.rs:7:3
  |
7 | /   a : Vec< T >,
8 | |   b : Vec< T >,
  | |_______________^
